python/triton/language/core.py

```diff
@@ -701,12 +701,20 @@ def get_int_dtype(bitwidth: int, signed: bool) -> dtype:
         raise ValueError(f'Unsupported bitwidth {bitwidth} and signedness {signed}')
 
 
+class _value:
+    """Base class of all triton values.
```

Comment:
I think we also have `const` and `constexpr`, are they also considered as "triton values"?